contacts_vs_distance.ipynb

```diff
@@ -44,7 +44,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cvds = []
</pre><p>would either be more clear as dictionary w/ downsample_frac as the key? or a dataFrame? or something else?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>